chef/main-cookbooks/acme/recipes/k8s.rb

```diff
@@ -0,0 +1,30 @@
+#
```

Comment:
please add support for "install" or "remove" action, refer here: https://github.com/iorad/dev-setup/pull/18